assets/js/main.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(() => {
```

Comment:
Why wait for document ready? The JS is injected at the end of the body, so the dom will be ready for manipulation at this point